PI-System-Deployment-Tests/History.md

```diff
@@ -5,4 +5,4 @@
 | 1.0.0   | Initial public release                |
 | 1.0.1   | Update for AF Client 2018 SP3 Patch 1 |
 | 1.0.2   | Update for PI Server 2018 SP3 Patch 1 |
-
+| 1.0.3   | Update for PI Manual Logger 2017 R2 Patch 2 (3.4.2.535) and PI Web API 2019 SP1 (1.13.0.6518) releases |
```

Comment:
Does it matter to us that they are tracking version history differently? (No dates, different file formatting)

Reply:
not to me...

Reply:
at some point it might be good to merge formatting, but we can worry about that next time